Cargo.toml

```diff
@@ -19,6 +19,14 @@ exclude = [
     "vendor/linux/**/libusb",
 ]
 
+[[bin]]
+name = "example_version"
+path = "examples/version.rs"
+
+[[bin]]
+name = "example_num_devices"
+path = "examples/num_devices.rs"
+
```

Comment:
Would it be better to add the examples to CI via `--examples` to the `cargo build` arguments?

I am not sure exposing the examples as binaries will have any unintended consequences for distribution.

Reply:
Very good catch.